aips/create_dip.py

```diff
@@ -411,15 +415,25 @@ def update_premis_ns(premis, namespaces, premis_map):
         )
 
 
-def get_original_name(premis, namespaces):
+def get_original_filename(premis, namespaces):
     """Get original filename from PREMIS record"""
+
     original_name = premis.findtext("premis:originalName", namespaces=namespaces)
     if not original_name:
         LOGGER.warning("premis:originalName could not be found")
-    string_start = "%transferDirectory%objects/"
-    if original_name[:27] != string_start:
-        LOGGER.warning("premis:originalName not starting with %s", string_start)
-    return original_name
+        return None
+
+    # Strip the path prefix from the file path, and return the remainder
+    path_prefixes = ["%transferDirectory%objects/", "%transferDirectory%data/"]
+    for pp in path_prefixes:
```

Comment:
Nit: We try to avoid single- (or double-) letter variable names. Could rename to `prefix`?

Reply:
Haha, I thought I was being all Pythonic by using a two letter variable name - most of the examples in the docs use single letter vars for temporary values from an iterator.  I'll switch it up.